### PR TITLE
OCPQE-6448: OCP-12564 improve and optimize the automation scripts

### PR DESCRIPTION
**Pull request by ShudiLi**
@lihongan, @quarterpin, request for a peer review
CC: @melvinjoseph86
thanks

### CODE REVIEW

File: features/routing/route.feature [hunk 1]
```diff
@@ -200,41 +200,15 @@ Feature: Testing route
...
-      | hostname   | <%= rand_str(5, :dns) %>-reen.example.com                                                     |
-      | service    | service-secure                                                                                |
-      | cert       | route_reencrypt-reen.example.com.crt |
-      | key        | route_reencrypt-reen.example.com.key |
-      | cacert     | route_reencrypt.ca                   |
-      | destcacert | route_reencrypt_dest.ca              |
-      | path       | /test                                                                                         |
+      | name       | route-recrypt           |
+      | service    | service-secure          |
+      | destcacert | route_reencrypt_dest.ca |
```
Comment (lihongan) [inline features/routing/route.feature:208]:
```
| path       | /test            |
```
should be kept since this case is testing the route with a path.

Comment (ShudiLi) [reply features/routing/route.feature:209]: Thanks for the review. I have added back the path, and modified the two checked points.

File: features/routing/route.feature [hunk 2]
```diff
@@ -199,42 +198,32 @@ Feature: Testing route
...
     When I run the :create_route_reencrypt client command with:
-      | name       | route-recrypt                                                                                 |
-      | hostname   | <%= rand_str(5, :dns) %>-reen.example.com                                                     |
-      | service    | service-secure                                                                                |
-      | cert       | route_reencrypt-reen.example.com.crt |
-      | key        | route_reencrypt-reen.example.com.key |
-      | cacert     | route_reencrypt.ca                   |
-      | destcacert | route_reencrypt_dest.ca              |
-      | path       | /test                                                                                         |
+      | path       | /test                   |
```
Comment (lihongan) [inline features/routing/route.feature:206]: Better to move this line/option to last, after the line `| destcacert | route_reencrypt_dest.ca |`

**Comment by lihongan** [comment]
@ShudiLi could you help update this case  `@author zzhao@redhat.com` to youself as well?

File: features/routing/route.feature (hunk 2 above)
Comment (ShudiLi) [reply features/routing/route.feature:206]: Ok, I will check and do it, thanks.

**Comment by ShudiLi** [comment]
@lihongan  sure, I will change the case author to myself

**Comment by ShudiLi** [comment]
cucushift automaton run:
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/211274/

**Comment by lihongan** [comment]
/lgtm
Thank you @ShudiLi

**Comment by ShudiLi** [comment]
@lihongan Thanks for the review and approve.

**Pull request merged**